--- utils.py.orig	2015-07-06 21:43:43.000000000 -0400
+++ utils.py	2015-07-06 21:43:53.000000000 -0400
@@ -186,7 +186,10 @@
     _translate_kwds = dict(checksum="hash", rounds="iterations")
 
     def safe_summary(self, encoded):
-        from django.contrib.auth.hashers import mask_hash, _, SortedDict
+        # from django.contrib.auth.hashers import mask_hash, _, SortedDict
+        from django.contrib.auth.hashers import mask_hash
+        from django.utils.translation import ugettext_noop as _
+        from django.utils.datastructures import SortedDict
         handler = self.passlib_handler
         items = [
             # since this is user-facing, we're reporting passlib's name,
